enterprise_catalog/apps/catalog/filters.py

```diff
@@ -0,0 +1,126 @@
+"""
+Utility functions for catalog query filtering without elasticsearch
+"""
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+SUPPORTED_FILTER_COMPARISONS = [
+    'exact',
+    'not',
+    'exclude',
+    'gt',
+    'gte',
+    'lt',
+    'lte',
+]
+
+
+class QueryFilterException(Exception):
+    """
+    An exception for content catalog query filtering
+    """
+
+
+def fix_common_query_key_mistakes(raw_query_key):
+    """
+    In production many queries have odd typos
+    which seem to have been copypasta-proliferated
+    """
+    raw_query_key = raw_query_key.lower()
+    if raw_query_key == 'aggregration__key':
+        return 'aggregation_key'
+    elif raw_query_key == 'aggregation__key':
+        return 'aggregation_key'
+    elif raw_query_key == 'org__exempt':
+        return 'org__exclude'
```

Comment:
nit: Could put this in a dict to reduce duplication and allow adding more cases without further stacking the elif tower
```suggestion
    corrections_for_typos = {
      'aggregation_key': ['aggregration__key', 'aggregation__key'],
      'org__exclude': ['org__exempt']
    }
    for correction, typos in corrections_for_typos.items():
      if raw_query in typos:
        return correction;
```